src/CMakeLists.txt

```diff
@@ -159,22 +161,25 @@ set(tfs_HDR
 	${CMAKE_CURRENT_LIST_DIR}/weapons.h
 	${CMAKE_CURRENT_LIST_DIR}/wildcardtree.h
 	${CMAKE_CURRENT_LIST_DIR}/xtea.h
+	${CMAKE_CURRENT_LIST_DIR}/logger/logger.h
 	)
 
 set(tfs_MAIN ${CMAKE_CURRENT_LIST_DIR}/main.cpp PARENT_SCOPE)
 
 add_library(tfslib ${tfs_SRC})
-target_link_libraries(tfslib PRIVATE
-	Boost::iostreams
-	Boost::system
+target_link_libraries(${PROJECT_NAME} PRIVATE
+	${MYSQL_CLIENT_LIBS}
+	${LUA_LIBRARIES}
+	PugiXML::pugixml
+	ZLIB::ZLIB
 	OpenSSL::Crypto
-	pugixml::pugixml
 	simdutf::simdutf
-	ZLIB::ZLIB
+	Threads::Threads
+	${Boost_LIBRARIES}
+	spdlog::spdlog
 	${CMAKE_THREAD_LIBS_INIT}
-	${LUA_LIBRARIES}
-	${MYSQL_CLIENT_LIBS}
 	)
```

Comment:
Please keep the alphabetical order, just add `spglog::spdlog` after `simdutf::simdutf`